Header header
int8 levelled
int8 state_of_trajectory
float64 loop_hz
